simple/mainTemplateParameters.json

```diff
@@ -15,7 +15,7 @@
      "value": 128
   },
   "graphDatabaseVersion": {
-    "value": "4.4.10"
+    "value": "5.1.0"
```

Comment:
Can you add the http version lookup from the script? There is an example how to do it the GCP marketplace repo

Reply:
thanks @ojhughes ...can you provide the link ?

Reply:
https://github.com/neo4j-partners/google-deployment-manager-neo4j/blob/main/simple/core-5.sh#L50